src/flags.rs

```diff
@@ -149,6 +153,7 @@ pub fn set_flags(
   opts.optflag("", "allow-env", "Allow environment access");
   opts.optflag("", "allow-run", "Allow running subprocesses");
   opts.optflag("A", "allow-all", "Allow all permissions");
+  opts.optflag("", "no-prompt", "Do not use prompts");
```

Comment:
--deny ? 
https://github.com/denoland/deno/issues/1580

Reply:
No prompt shouldn't be limited to permissions, so in that respect it's different than `--deny`. Either your there to answer for prompts or you aren't as a user.

Reply:
What other prompts are there?

I don't understand the difference.

Reply:
--deny would be confusing when combined with, say, --allow-read
```
> deno  foo.ts --allow-read --deny
```
What is intended is: "run deno with foo.ts as the main script, allow fs read access, but do not prompt for permission"
But people might interpret this as: run deno with foo.ts as the main script, allow fs read access, deny all access (overriding previous?)"

Reply:
--no-prompt is kinda long.... Can we quickly just throw out a few other suggestions?

Reply:
No other suggestions it seems. I can't think of anything better...

Reply:
what about `--no` (as analogous to `--yes` e.g. of unix tools)